inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/utils/ProxyUtils.java

```diff
@@ -92,6 +92,46 @@ public static boolean isBodyValid(List<byte[]> bodyList) {
         return true;
     }
 
+    /**
+     * Check if the body length exceeds the maximum limit, if the maximum limit is less than or equal to 0, it is not checked
+     * @param body
+     * @param maxLen
+     * @return
+     */
+    public static boolean bodyLengthCheck(byte[] body, int maxLen) {
+        // Not valid if the maximum limit is less than or equal to 0
+        if (maxLen <= 0) {
+            return true;
+        }
+        if (body.length > maxLen) {
+            logger.error("body length is too long, max length is {}", maxLen);
```

Comment:
The failure logs for messages should be at debug level to avoid excessive junk logs.